core-web/libs/edit-content/src/lib/components/dot-edit-content-field/dot-edit-content-field.component.ts

```diff
@@ -5,16 +5,21 @@ import { ControlContainer, ReactiveFormsModule } from '@angular/forms';
 import { DotCMSContentTypeField } from '@dotcms/dotcms-models';
 import { DotFieldRequiredDirective } from '@dotcms/ui';
 
+import { FIELD_TYPES } from './utils';
+
+import { DotEditContentFieldsModule } from '../../fields/dot-edit-content-fields.module';
 import { DotEditContentRadioFieldComponent } from '../../fields/dot-edit-content-radio-field/dot-edit-content-radio-field.component';
 import { DotEditContentSelectFieldComponent } from '../../fields/dot-edit-content-select-field/dot-edit-content-select-field.component';
 
 @Component({
     selector: 'dot-edit-content-field',
     standalone: true,
     imports: [
-        CommonModule,
+        NgSwitch,
+        NgSwitchCase,
+        NgIf,
         ReactiveFormsModule,
-        InputTextModule,
+        DotEditContentFieldsModule,
         DotFieldRequiredDirective,
         DotEditContentSelectFieldComponent,
         DotEditContentRadioFieldComponent
```

Comment:
remove from here and import and export these in the `DotEditContentFieldsModule`